product_docs/docs/tde/15/enabling_tde.mdx

```diff
@@ -65,7 +65,8 @@ If you don't specify this parameter, the environment variable `PGDATAKEYUNWRAPCM
 
 Use the special value `-` if you don't want to apply any key unwrapping command.
 
-<!--Should this be worded differently? Is this the environment variable fallback? I changed the word "option" to parameter. -->
+<!--- Should this be worded differently? 
```

Comment:
I think we will be removing this and the following line

Reply:
Removed... No idea how the line break appeared, but we should resolve questions about content without leaving long lived and untracked comments in the docs.